src/app/components/core/configuration-manager/view-config-item/view-config-item.component.html

```diff
@@ -39,45 +39,59 @@
                       [attr.maxLength]="configItem?.value?.length" [jsonValue]="setConfigValue(configItem.value)"
                       required>{{configItem?.value?.value != undefined ? configItem?.value?.value : configItem?.value?.default}}</textarea>
 
+                    <textarea rows="3" *ngIf="getConfigAttributeType(configItem.value.type) === 'SCRIPT'" class="textarea is-fullwidth is-small"
+                      name="{{configItem.value.key}}" [ngModel]="setConfigValue(configItem.value)" [value]='setConfigValue(configItem.value)'
+                      [attr.maxLength]="configItem?.value?.length">{{configItem?.value?.value != undefined ? configItem?.value?.value : configItem?.value?.default}}</textarea>
+
                     <div *ngIf="getConfigAttributeType(configItem.value.type) == 'ENUMERATION'" class="select is-small">
                       <select name='{{configItem.value.key}}' [ngModel]="setConfigValue(configItem.value)">
                         <option *ngFor="let configOption of configItem.value.options" [value]='configOption'>{{configOption}}</option>
                       </select>
                     </div>
+
+                    <div *ngIf="getConfigAttributeType(configItem.value.type) == 'SCRIPT'">
+                      <div class="config-div is-fullwidth">
+                        <input class="input is-small is-fullwidth" name="{{configItem.value.key}}" type="file"
+                          #fileInput />
+                        <a class="button is-small is-text is-pulled-right" (click)="uploadScript(configItem.value.key)">upload</a>
```

Comment:
please remove this upload button, and let it be called when `Save` button is clicked as for other config items